README.md: let tiup be the only playground method (#10881)
tikv/tikv: README.md

- **Consistent distributed transactions**: Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions.
- **Coprocessor support**: Similar to HBase, TiKV implements a coprocessor framework to support distributed computing.
- **Cooperates with [TiDB](https://github.com/pingcap/tidb)**: Thanks to the internal optimization, TiKV and TiDB can work together to be a compelling database solution with high horizontal scalability, externally-consistent transactions, support for RDBMS, and NoSQL design patterns.

![cncf_logo](images/cncf.png)

TiKV is a graduated project of the [Cloud Native Computing Foundation](https://cncf.io/) (CNCF). If you are an organization that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how TiKV plays a role, read the CNCF [announcement](https://www.cncf.io/announcements/2020/09/02/cloud-native-computing-foundation-announces-tikv-graduation/).

## TiKV Adopters

You can view the list of [TiKV Adopters](https://tikv.org/adopters/).

## Quick Start

> The most quickest to try out TiKV is using TiUP:

Reviewer: ```suggestion
The quickest way to try out TiKV is using TiUP, an automation program for downloading, updating, and uninstalling TiDB/TIKV components:
```

Author: I intentionally removed the description of TiUP since it's already famous enough, how about only leaving a link to TiUP?

```shell
# install tiup
curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh

# deploy a TiKV playground
tiup playground --mode tikv-slim
```

> You can see [TiKV in 5 Minutes](https://tikv.org/docs/5.1/concepts/tikv-in-5-minutes/) for a step by step tutorial.

Reviewer: ```suggestion
Watch [TiKV in 5 Minutes](https://tikv.org/docs/5.1/concepts/tikv-in-5-minutes/) for a step-by-step tutorial.
```

Author: it's a document, maybe `see` is more suitable?

> For productization deployment, please refer to [deploy](https://tikv.org/docs/5.1/deploy/deploy/) for details. You can also see [this manual](./doc/deploy.md) of production-like cluster deployment presented by @c4pt0r.

Reviewer: ```suggestion
For deployment in your production environment, please refer to [deploy](https://tikv.org/docs/5.1/deploy/deploy/) for details. You can also see [this manual](./doc/deploy.md) of production-like cluster deployment presented by @c4pt0r.
```

## Client Drivers

> Currently, the most mature TiKV clients are the [Go](https://github.com/tikv/client-go) and [Java](https://github.com/tikv/client-java) clients, clients for other languages are under development. All the supported clients are:

Reviewer: ```suggestion
Currently, the most stable TiKV clients are the [Go](https://github.com/tikv/client-go) and [Java](https://github.com/tikv/client-java) clients. Clients for other languages are under development. The complete list of supported clients are:
```

> **Note:**
>
> We have migrated our documentation from the [TiKV's wiki page](https://github.com/tikv/tikv/wiki/) to the [official website](https://tikv.org/docs). The original Wiki page is discontinued. If you have any suggestions or issues regarding documentation, offer your feedback [here](https://github.com/tikv/website).

## TiKV Architecture

![The TiKV software stack](images/tikv_stack.png)

TiKV is composed of:

- **Placement Driver:** PD is the cluster manager of TiKV, which periodically checks replication constraints to balance load and data automatically.
- **Store:** There is a RocksDB within each Store and it stores data into the local disk.
- **Region:** Region is the basic unit of Key-Value data movement. Each Region is replicated to multiple Nodes. These multiple replicas form a Raft group.
- **Node:** A physical node in the cluster. Within each node, there are one or more Stores. Within each Store, there are many Regions.

When a node starts, the metadata of the Node, Store and Region are recorded into PD. The status of each Region and Store is reported to PD regularly.

## TiKV Community

### Contribute to TiKV

> The design of TiKV is inspired by some great distributed systems from Google, such as BigTable, Spanner, and Percolator, and some of the latest achievements in academia in recent years, such as the Raft consensus algorithm. Built in Rust and powered by Raft, TiKV was originally created to complement [TiDB](https://github.com/pingcap/tidb), a distributed HTAP database compatible with the MySQL protocol.

Reviewer: ```suggestion
The design and implementation of TiKV is inspired by some great distributed systems from Google, such as BigTable, Spanner, and Percolator, and some of the latest achievements in academia such as the Raft consensus algorithm. Built in Rust and powered by Raft, TiKV was originally created to complement [TiDB](https://github.com/pingcap/tidb), a distributed HTAP database compatible with the MySQL protocol.
```

> If you're interested in contributing to TiKV, or want to build it from source, see [CONTRIBUTING.md](./CONTRIBUTING.md).

Reviewer: ```suggestion
If you're interested in contributing to TiKV or want to build it from the source, see [CONTRIBUTING.md](./CONTRIBUTING.md).
```

We will invite you in right away.

## Security

### Security Audit

> A third-party security auditing was performed by Cure53. See the full report [here](./security/Security-Audit.pdf).

Reviewer: ```suggestion
A CNCF sponsored third-party security auditing was performed by Cure53. See the full report [here](./security/Security-Audit.pdf).
```
